ComponentKit/Core/CKCompositeComponent.mm

```diff
@@ -64,6 +65,33 @@ + (instancetype)newWithView:(const CKComponentViewConfiguration &)view size:(con
   CK_NOT_DESIGNATED_INITIALIZER();
 }
 
+- (CKComponent *)render:(id)state
+{
+  return _component;
```

Comment:
I've decided to implement the render method here in order to allow mix-and-match of the "old" components with the "new" ones. Otherwise, we might have a `CKCompositeComponent` in the component tree; in this case it will not continue to call `buildComponentTree` recursively.